README.md

```diff
@@ -1,10 +1,26 @@
 # bazel-aarch64-linux-gnu
 This repo provides a Bazel cross toolchain for x86_64/Linux host and aarch64/Linux target
 
-The structure was inspired by: https://asnaghi.me/post/embedded-bazel/
+The structure was inspired by: https://asnaghi.me/post/embedded-bazel/,
+then re-worked to support bazel CC toolchain auto-resolution / platforms.
 
 ## Toolchain
-The toolchain implemented is : `gcc-arm-9.2-2019.12-x86_64-aarch64-none-linux-gnu`
+The toolchain implemented is: gcc-11-aarch64-linux-gnu_11.4.0, from Ubuntu 22.04
```

Comment:
Should we enumerate all three toolchains? aarch64 cross, aarch64 native, and x86_64 native?